### ISSUE TITLE
Selector issues

### ISSUE THREAD

**Issue by charlieMonroe** [drkameleon/iswift#114]
Here's a few selectors in use:

```
SEL selector1 = @selector(hello);
SEL selector2 = @selector(hello:didSomething:);
```

This gets converted to:

```
let selector1: SEL = hello
let selector2: SEL = hello,didSomething
```

which isn't right. The correct type is `Selector` instead of `SEL` and should look like this:

```
let selector1: Selector = "hello"
let selector2: Selector = "hello:didSomething:"
```

Note that it is correct to "assign" `String` to `Selector`, since it has an initializer that takes `String` as an argument, hence automatic conversion takes in place.

**Comment by drkameleon** [comment]
Question:

Aside for assigning a string to `Selector`, what about using it in place of all selectors (e.g. as a param). Would a string work there as well?

**Comment by drkameleon** [comment]
Issue fixed as of the upcoming 2.0.

----

**Example:**

```objective-c
@implementation aClass

- (void)someFunc {
    SEL selector1 = @selector(hello);
    SEL selector2 = @selector(hello:didSomething:);
}

@end
```

**Output:**

```swift
class aClass {

	func someFunc() {
		var selector1: Selector = "hello"
		var selector2: Selector = "hello:didSomething:"
	
	}
	
}
```
